ts/sagas/startup/checkProfileEnabledSaga.ts

```diff
@@ -10,17 +10,11 @@ export function* checkProfileEnabledSaga(
   profile: InitializedProfile
 ): IterableIterator<Effect> {
   if (
-    !profile.has_profile ||
+    profile.version === 0 ||
```

Comment:
I guess we can split this in 2 blocks.
A block checks for **is_inbox_enabled** and **is_webhook_enabled** (as it is we could call an upsert when it is not needed)
Another block check if `profile.version === 0` and if yes dispatch the action

Also the tests inside **_checkProfileEnabledSaga.test.ts_** should be updated

Reply:
Yes, I was checking the CircleCI script right now